src/libraries/System.Private.CoreLib/src/System/Decimal.cs

```diff
@@ -20,8 +20,8 @@ namespace System
     // require a large number of significant digits and no round-off errors.
     //
     // The finite set of values of type Decimal are of the form m
-    // / 10e, where m is an integer such that
-    // -296 <; m <; 296, and e is an integer
+    // / 10^e, where m is an integer such that
+    // -2^96 <; m <; 2^96, and e is an integer
```

Comment:
```suggestion
    // -2^96 <= m <= 2^96, and e is an integer
```

Is there something I'm missing about the use of `;` here?

Reply:
This comment got corrupted in June 2002. Originally it looked this way:
```
     * <p>The finite set of values of type <i>Decimal</i> are of the form <i>m</i>
     * / 10<sup><i>e</i></sup>, where <i>m</i> is an integer such that
     * -2<sup>96</sup> &lt; <i>m</i> &lt; 2<sup>96</sup>, and <i>e</i> is an integer
     * between 0 and 28 inclusive.
```
You should use `<`, not `<=`. Also please move `/ 10^e` to the previous line.